components/database/couchdb-21/files/couchdb-21.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+
+This file and its contents are supplied under the terms of the
+Common Development and Distribution License ("CDDL)". You may
+only use this file in accordance with the terms of the CDDL.
+
+A full copy of the text of the CDDL should have accompanied this
+source. A copy of the CDDL is also available via the Internet at
+http://www.illumos.org/license/CDDL.
+
+Copyright 2012 EveryCity Ltd. All rights reserved.
+
+-->
+<service_bundle type="manifest" name="couchdb">
+
+  <service name="application/database/couchdb-21" type="service" version="1">
+
+    <dependency name="network" grouping="require_all" restart_on="error" type="service">
+      <service_fmri value="svc:/milestone/network:default"/>
+    </dependency>
+
+    <dependency name="filesystem-local" grouping="require_all" restart_on="none" type="service">
+      <service_fmri value="svc:/system/filesystem/local:default"/>
+    </dependency>
+
+		<exec_method type="method" name="start" exec="/usr/lib/amd64/couchdb-21/bin/couchdb &amp;" timeout_seconds="60">
```

Comment:
Try to avoid using amd64|sparcv9 in manifests explicitly.

Reply:
replaced by adding links to usr/bin/